### PR TITLE
Fixing the ElseIf activity arrow flow in bpel process visualization

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java [hunk 1]
```diff
@@ -93,28 +90,29 @@ public void setParent(ActivityInterface parent) {
...
     protected int boxXLeft = 0;
     protected int boxYTop = 0;
     protected int boxHeight = 0;
     protected int boxWidth = 0;
-    protected String boxStyle = "fill-opacity:0.04;fill-rule:evenodd;stroke:#0000FF;stroke-width:1.99999988;" +
-            "stroke-linecap:square;stroke-linejoin:bevel;stroke-miterlimit:1;stroke-dasharray:none;" +
+    protected String boxStyle = "fill-opacity:0.04;fill-rule:evenodd;stroke:#0000FF;stroke-width:1.99999988;"
+            +
+            "stroke-linecap:square;stroke-linejoin:bevel;stroke-miterlimit:1;stroke-dasharray:none;"
+            +
```
Comment (jranabahu) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Please format accordingly

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java [hunk 2]
```diff
@@ -129,7 +127,8 @@ public ActivityImpl(String token) {
             if (firstQuoteIndex >= 0) {
                 int lastQuoteIndex = token.indexOf("\"", firstQuoteIndex + 1);
                 if (lastQuoteIndex > firstQuoteIndex) {
-                    setName(token.substring(firstQuoteIndex + 1, lastQuoteIndex));
+                    setName(token
+                            .substring(firstQuoteIndex + 1, lastQuoteIndex));
```
Comment (jranabahu) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Please format accordingly

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java [hunk 3]
```diff
@@ -550,46 +554,54 @@ protected Element getImageDefinition(SVGDocument doc, String imgPath, int imgXLe
...
+            text1.setAttributeNS(null, "style",
+                    "font-size:12px;font-style:normal;font-variant:normal;font-weight:"
+                            +
+                            "normal;font-stretch:normal;text-align:start;line-height:125%;writing-mode:lr-tb;text-anchor:"
+                            +
+                            "start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;"
+                            +
+                            "stroke-linejoin:bevel;stroke-opacity:1;font-family:Arial Narrow;"
+                            +
+                            "-inkscape-font-specification:Arial Narrow");
```
Comment (jranabahu) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Please format correctly

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java [hunk 4]
```diff
@@ -638,12 +657,18 @@ protected String getArrowStyle() {
...
+                        +
+                        "butt;stroke-linejoin:bevel;marker-end:url(#LinkArrow);stroke-dasharray:"
+                        +
+                        "none;stroke-opacity:1;opacity: 0.25;";
+        String mediumArrowStr =
+                "fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:1.5;stroke-linecap:"
+                        +
+                        "butt;stroke-linejoin:bevel;marker-end:url(#LinkArrow);stroke-dasharray:"
+                        +
+                        "none;stroke-opacity:1;opacity: 0.25;";
```
Comment (jranabahu) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: The String formatting needs to be fixed

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java [hunk 5]
```diff
@@ -674,20 +699,25 @@ protected Element getArrowDefinition(SVGDocument doc, int startX, int startY, in
...
-        path.setAttributeNS(null, "d", "M " + startX + "," + startY + " L " + midX + "," + midY + "L " + endX +
-                "," + endY);
+    protected Element getArrowDefinition(SVGDocument doc, int startX,
+                                         int startY, int midX, int midY, int endX, int endY, String id) {
+        Element path = doc
+                .createElementNS("http://www.w3.org/2000/svg", "path");
+        path.setAttributeNS(null, "d",
+                "M " + startX + "," + startY + " L " + midX + "," + midY + "L "
+                        + endX +
+                        "," + endY);
```
Comment (jranabahu) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Fix formatting

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java (hunk 1 above)
Comment (NatashaWso2) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Fixed with commit 226ec4f560fcbb6964e0ad72e9b915e2509b62af

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java (hunk 2 above)
Comment (NatashaWso2) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Fixed with commit 226ec4f560fcbb6964e0ad72e9b915e2509b62af

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java (hunk 3 above)
Comment (NatashaWso2) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Fixed with commit 226ec4f560fcbb6964e0ad72e9b915e2509b62af

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java (hunk 4 above)
Comment (NatashaWso2) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Fixed with commit 226ec4f560fcbb6964e0ad72e9b915e2509b62af

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java (hunk 5 above)
Comment (NatashaWso2) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Fixed with commit 226ec4f560fcbb6964e0ad72e9b915e2509b62af

**Pull request merged**